mmdet/models/utils/gaussian_target.py

```diff
@@ -54,7 +54,7 @@ def gen_gaussian_target(heatmap, center, radius, k=1):
     masked_heatmap = heatmap[y - top:y + bottom, x - left:x + right]
     masked_gaussian = gaussian_kernel[radius - top:radius + bottom,
                                       radius - left:radius + right]
-    out_heatmap = torch.zeros_like(heatmap)
+    out_heatmap = heatmap
```

Comment:
Will this change the input `heatmap`? Is this behavior expected or not?

Reply:
The `torch.max` will change the input `heatmap`, and we would also return the changed `heatmap`, so we didn't clone the `heatmap` here to save cost and this behavior is expected.

Reply:
The `torch.max` will change the input `heatmap`, and we would also return the changed `heatmap`, so we didn't clone the `heatmap` here to save cost and this behavior is expected.